tests/testthat/test-git_tools.R

```diff
@@ -0,0 +1,24 @@
+test_that("checkout_repo works", {
+  # sodo: mock git clone etc so it works in automation (will be addressed in issue #33)
```

Comment:
Normally I wouldn't commit this but if it's going to be replaced soon then that's probably ok (also typo "sodo:"